### ISSUE TITLE
No nos da logros al  hacer masmorra aleatoria  con la hermandad 

### ISSUE THREAD

**Issue by deiver9696** [RoDNet/Bugtracker-Infinium-4.0.6a#441]
Tenemos un problema que al hacer masmorras aleatorias con la hermandad no dan logro ni nada entonces como subimos de nivel la hermandad el dado caso es que no  la hemos podido pasar de nivel uno espero que nos ayuden pronto 
![Imagen2](https://f.cloud.github.com/assets/3053230/80449/f802c598-625b-11e2-9dec-e93c162f47db.png)
![Imagen1](https://f.cloud.github.com/assets/3053230/80450/1180874e-625c-11e2-905c-484155c78f7a.png)

**Comment by asfo** [comment]
Corríganme si me equivoco pero, no debería de dar logros así, únicamente si se hacen de manera tradicional.

**Comment by deiver9696** [comment]
Y cual es esa manera tradicional

**Comment by DatJetto** [comment]
No, Asfo, debería dar Logro>   Heroico/Normal: Hermandad: X mazmorra

**Comment by asfo** [comment]
Es que yo recuerdo que en el oficial la mazmorra aleatoria ignoraba si era guild o no y tenias que ir hasta la mazmorra y entrar como de costumbre...

**Comment by deiver9696** [comment]
Bueno pero que me dices de kael? no dio logro

**Comment by DatJetto** [comment]
En el oficial claro que lo da, es el nuevo sistema cata de logros de hermandad si es lo más innovador de cata! 
Multi, alomejor ya tu guild tiene el logro y por eso no os salta.

**Comment by deiver9696** [comment]
No tenemos logro de ninguna de esas   ni   de las masmorras de nivel bajo  no nos da cuando vamos para subir  de nivel la hermandad

**Comment by asfo** [comment]
@DatJetto Mi hermandad iba en aleatorias seguido por que si vas entre 3 o más te da XP extra por haber completado pero no los logros, los logros los tenías qué hacer de forma tradicional llendo los 5 de la hermandad a hacer la mazmorra. Si no mal recuerdo para la 4.3.4 se había cambiado y ya se podía con las aleatorias, pero en la 4.0.6 nos pasó eso...Debo comprobar en algún foro eso ;) y así lo anexo para darme cuenta que me drogo xD y estoy mal...

Mientras tanto, viendo lo de Kael'thas debió darles logo por que esa no entra en las "aleatorias", se estará entonces reparando esto ;).
